site/src/components/Pagination/Pagination.types.ts

```diff
@@ -0,0 +1,7 @@
+export type Props = {
```

Comment:
Have you already defined this?

Reply:
most fields are the same as Pagination from usePagination
it can be written using Omit<> or Pick<>. What's better?

Reply:
I prefer using Omit<> or Pick<>